source/api/extension/navplace/index.md

```diff
@@ -0,0 +1,331 @@
+* __HTTP(S)__: The HTTP or HTTPS URI scheme and internet protocol.
+* __location__: the quantitative description of the position of a place, in this case, using coordinates.
+* __GeoJSON__: "The GeoJSON Format" specification published at [https://datatracker.ietf.org/doc/html/rfc7946](https://datatracker.ietf.org/doc/html/rfc7946).
+* __GeoJSON-LD__: The Linked Data implementation of the GeoJSON specification published at [https://geojson.org/geojson-ld/](https://geojson.org/geojson-ld/).
+* __linked data context__: A reference to a [JSON-LD context file](https://www.w3.org/TR/json-ld11/#the-context) as described by JSON-LD 1.1
+
+The terms _array_, _JSON object_, _number_, _string_, and _boolean_ in this document are to be interpreted as defined by the [Javascript Object Notation (JSON)][org-rfc-8259] specification.
+
+The key words _MUST_, _MUST NOT_, _REQUIRED_, _SHALL_, _SHALL NOT_, _SHOULD_, _SHOULD NOT_, _RECOMMENDED_, _MAY_, and _OPTIONAL_ in this document are to be interpreted as described in [RFC 2119][org-rfc-2119].
+
+
+## 2. The `navPlace` Property and GeoJSON-LD
+
+
+### 2.1 `navPlace` Property
+
+The `navPlace` property identifies a single or multiple geographic areas pertinent to a resource using a GeoJSON Feature Collection containing one or more Features. These areas _SHOULD_ be bounded discrete areas of the map akin to extents. These areas do not imply any level of accuracy, temporality, or state of existence. 
+
+
+```json-doc
+{
+   "navPlace":{
+      "id": "http://example.com/feature-collection/1",
+      "type": "FeatureCollection",
+      "features":[
+         {
+            "id": "http://example.com/feature/1",
+            "type": "Feature",
+            "properties":{},
+            "geometry":{
+               "type": "Point",
+               "coordinates":[
+                  9.938,
+                  51.533
+               ]
+            }
+         }
+      ]
+   }
+}
+```
+
+
+The `navPlace` property can be used with the [IIIF Presentation 3 API Defined Types](https://iiif.io/api/presentation/3.0/#21-defined-types).
+
+
+
+*   A Collection _MAY_ have the `navPlace` property.<br/>
+   Clients _MAY_ render `navPlace` on a Collection.
+*   A Manifest _MAY_ have the `navPlace` property.<br/>
+   Clients _MAY_ render `navPlace` on a Manifest.
+*   A Range _MAY_ have the `navPlace` property.<br/>
+   Clients _MAY_ render `navPlace` on a Range.
+*   A Canvas _MAY_ have the `navPlace` property.<br/>
+   Clients _MAY_ render `navPlace` on a Canvas.
+*   Other types of resource _MUST NOT_ have the `navPlace` property.<br/>
+   Clients _SHOULD_ ignore `navPlace` on other types of resource.
+
+The `navPlace` property's value follows a specific pattern.
+
+
+
+*   The value _SHOULD_ be an embedded Feature Collection. However, the value _MAY_ be a referenced GeoJSON Feature Collection. 
```

Comment:
Does this mean the 'type' value? Feels vague. Or should this be 'values' (plural)?